src/lib/codemirror/init.ts

```diff
@@ -44,76 +85,105 @@ export const initialExtensions = (onUpdate: any, onRunShortcut: any, yCollab?: a
 	search({
 		top: true,
 		createPanel(view) {
-			const dom = document.createElement('div')
-			const query = signal(getSearchQuery(view.state))
-			const cursor = signal({ index: 0, count: 0 })
+			const dom = document.createElement("div");
+			const query = signal(getSearchQuery(view.state));
+			const cursor = signal({ index: 0, count: 0 });
 
-			let _firstUpdate = true
+			let _firstUpdate = true;
 			effect(() => {
-				const update = view.state.update({ effects: setSearchQuery.of(query.value) })
+				const update = view.state.update({
+					effects: setSearchQuery.of(query.value),
+				});
 				if (_firstUpdate) {
-					_firstUpdate = false
-					return
+					_firstUpdate = false;
+					return;
 				}
-				view.dispatch(update)
-			})
+				view.dispatch(update);
+			});
 
-			render(h(SearchBox, {
-				query,
-				cursor,
-				runCommand(command) { command(view) }
-			}), dom)
+			render(
+				h(SearchBox, {
+					query,
+					cursor,
+					runCommand(command) {
+						command(view);
+					},
+				}),
+				dom
+			);
 
 			return {
 				dom,
 				update(update) {
-					query.value = getSearchQuery(update.state)
+					query.value = getSearchQuery(update.state);
 
-					let [index, count] = [0, 0]
+					let [index, count] = [0, 0];
 					if (query.value.valid) {
-						const iter = query.value.getCursor(update.state)
-						for (let item = iter.next(); !item.done; item = iter.next()) {
-							count++
-							if (item.value.from <= update.state.selection.main.from && item.value.to >= update.state.selection.main.to)
-								index = count
+						const iter = query.value.getCursor(update.state);
+						for (
+							let item = iter.next();
+							!item.done;
+							item = iter.next()
+						) {
+							count++;
+							if (
+								item.value.from <=
+									update.state.selection.main.from &&
+								item.value.to >= update.state.selection.main.to
+							)
+								index = count;
```

Comment:
Please undo the formatting changes. It makes it hard to read what has actually changed.